site/project/who.md

```diff
@@ -41,6 +42,8 @@ ivank | Ivan Kelly | Midokura | +2
 jiannan	| Jiannan Wang | Yahoo Inc. | +8
 jujjuri	| Venkateswararao (JV) Jujjuri | Salesforce | -8
 mmerli | Matteo Merli | Streamlio | -8
+nicoloboschi | Nicolò Boschi | Datastax | +2
+prashantkumar | Prashant Kumar | Splunk | -8
```

Comment:
there is another prashantkumar at line 52/55 (end of the list) - please remove that one, it is out of alphabetical order. site3/ version updates it correctly.